src/functions/module_lookup.ts

```diff
@@ -834,8 +847,16 @@ export const PIVOT = {
       result[col] = [];
       for (const row of tableRows) {
         const pivotCell = cells[col][row];
-        const fn = pivotCell.isHeader ? PIVOT_HEADER : PIVOT_VALUE;
-        result[col].push(getPivotCellValueAndFormat.call(this, _pivotFormulaId, pivotCell, fn));
+        if (!pivotCell.domain) {
+          result[col].push({ value: "", format: undefined });
+        } else if (pivotCell.isHeader) {
+          result[col].push(pivot.getPivotHeaderValueAndFormat(pivotCell.domain));
+        } else {
+          if (!pivotCell.measure) {
+            throw new Error("Measure is missing");
```

Comment:
is this a user-facing error ? If yes, it should be a `EvaluationError` and the message should be translated.
if not, can't we fallback on an empty string and let `getPivotHeaderValueAndFormat` manage the error? or use `assertMeasureExist`

Reply:
The typing of `SPTableCell` is not really correct has, if `isHeader` is false, `measure` is required. So this message is a dev error. But the proper fix is to fix the typing, I will do it.

Reply:
1. Do not change anything